itests/hive-unit/pom.xml

```diff
@@ -116,6 +116,32 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
this dependency seems to be used in itests/hive-unit and ql too, questions:
1. would it make sense to define this in root pom's dependencyManagement with exclusions and just refer to that here?
2. the exclusions are different in the two places, is that intentional?

Reply:
1.) I followed the same pattern done for other tez dependencies line tez-dag etc. In parent pom.xml, just dependency in there and in ql/pom.xml, they have done the exclusion. Same goes for itests module and sub modules.

2.) Every tez dependency is trying to exclude some hadoop dependencies, but they are not same in submodules, for example, https://github.com/apache/hive/blob/master/itests/qtest/pom.xml#L392 and https://github.com/apache/hive/blob/master/itests/hive-minikdc/pom.xml#L150, both of then are tez-dag dependecy but have different exclusions. Hence I don't know which hadoop dependecy to exclude, and what's the history about them. That's why I kept the exclusions same as other tez dependencies in same pom.xml

If there is any suggestion around it, I am happy to re-evaluate and re work on the exclusions.

Reply:
@abstractdog, apologies for delay. Can you provide your guidance on this?

Reply:
@Aggarwal-Raghav can you pull this to the parent pom as Laszlo suggested & for the exclusions, combine all of them together, do check once if they are actually being pulled in or not.

if this approach creates some issue, we can consider excluding dependencies individually